pkg/api/plugins.go

```diff
@@ -21,35 +20,37 @@ import (
 // ErrPluginNotFound is returned when an requested plugin is not installed.
 var ErrPluginNotFound error = errors.New("plugin not found, no installed plugin with that id")
 
-func (hs *HTTPServer) getPluginConfig(pluginID string, user *models.SignedInUser) (backendplugin.PluginConfig, error) {
-	pluginConfig := backendplugin.PluginConfig{}
+func (hs *HTTPServer) getPluginContext(pluginID string, user *models.SignedInUser) (backend.PluginContext, error) {
+	pc := backend.PluginContext{}
 	plugin, exists := plugins.Plugins[pluginID]
 	if !exists {
-		return pluginConfig, ErrPluginNotFound
+		return pc, ErrPluginNotFound
 	}
 
-	var jsonData *simplejson.Json
-	var decryptedSecureJSONData map[string]string
-	var updated time.Time
+	// var jsonData *simplejson.Json
+	// var decryptedSecureJSONData map[string]string
+	// var updated time.Time
 
 	ps, err := hs.getCachedPluginSettings(pluginID, user)
-	if err != nil {
-		if err != models.ErrPluginSettingNotFound {
-			return pluginConfig, errutil.Wrap("Failed to get plugin settings", err)
-		}
-		jsonData = simplejson.New()
-		decryptedSecureJSONData = make(map[string]string)
-	} else {
-		decryptedSecureJSONData = ps.DecryptedValues()
-		updated = ps.Updated
-	}
-
-	return backendplugin.PluginConfig{
-		OrgID:                   user.OrgId,
-		PluginID:                plugin.Id,
-		JSONData:                jsonData,
-		DecryptedSecureJSONData: decryptedSecureJSONData,
-		Updated:                 updated,
+	_, _ = ps, err
+	// if err != nil {
+	// 	if err != models.ErrPluginSettingNotFound {
+	// 		return nil, errutil.Wrap("Failed to get plugin settings", err)
+	// 	}
+	// 	jsonData = simplejson.New()
+	// 	decryptedSecureJSONData = make(map[string]string)
+	// } else {
+	// 	decryptedSecureJSONData = ps.DecryptedValues()
+	// 	updated = ps.Updated
+	// }
+
+	return backend.PluginContext{
+		OrgID:    user.OrgId,
+		PluginID: plugin.Id,
+		// TODO: Which thing?
```

Comment:
`AppInstanceSettings`, this is for `/api/plugins/<plugin id>/<metrics|health|resources>` endpoints